keyboards/kingly_keys/smd_milk/readme.md

```diff
@@ -0,0 +1,21 @@
+# 2% Milk
+
+![SMD_MILK](https://cdn.discordapp.com/attachments/527180441950617615/628391976001208360/image0.jpg)
```

Comment:
Might be better to put this on imgur instead, I don't know how long Discord keeps images around for.